src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -549,25 +560,26 @@
   [message-content-wrapper message
    [unknown-content-type message]])
 
-(defn chat-message [{:keys [outgoing display-photo? pinned pinned-by] :as message} space-keeper]
-  [:<>
-   [reactions/with-reaction-picker
-    {:message         message
-     :reactions       @(re-frame/subscribe [:chats/message-reactions (:message-id message) (:chat-id message)])
-     :picker-on-open  (fn []
-                        (space-keeper true))
-     :picker-on-close (fn []
-                        (space-keeper false))
-     :send-emoji      (fn [{:keys [emoji-id]}]
-                        (re-frame/dispatch [::models.reactions/send-emoji-reaction
-                                            {:message-id (:message-id message)
-                                             :emoji-id   emoji-id}]))
-     :retract-emoji   (fn [{:keys [emoji-id emoji-reaction-id]}]
-                        (re-frame/dispatch [::models.reactions/send-emoji-reaction-retraction
-                                            {:message-id        (:message-id message)
-                                             :emoji-id          emoji-id
-                                             :emoji-reaction-id emoji-reaction-id}]))
-     :render          ->message}]
-   (when pinned
-     [react/view {:style (style/pin-indicator-container outgoing)}
-      [pinned-by-indicator outgoing display-photo? pinned-by]])])
+(defn chat-message [{:keys [outgoing display-photo? pinned pinned-by hidden-in-ui?] :as message} space-keeper]
+  (when-not hidden-in-ui?
```

Comment:
why don't we filter it in the subscription?

Reply:
Got rid of this.